app/(auth)/sign-in/page.tsx

```diff
@@ -1,21 +1,26 @@
 "use client"
 
-import { useAuthContext } from "@/context/auth"
```

Comment:
@tranduybau I think we've different about prettier here. Can you setup prettier for uniformity?
`
// from
import { useAuthContext }
// to
import {useAuthContext}
`
or vice versa